src/mixins/filing-template-mixin.ts

```diff
@@ -414,6 +415,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // restore Business Number
     this.setRegistrationBusinessNumber(draftFiling.registration.business.taxId || null)
 
+    // restore Business Type Confirm
+    this.setRegistrationBusinessTypeConfirm(draftFiling.registration.businessTypeConfirm)
```

Comment:
In the latest commit 5511d94, businessTypeConfirm uses the initial value from state-model, which is false, for both SP and GP.

Reply:
Yes, businessTypeConfirm in the store is initialized correctly.

However, here a draft filing is loaded. For a brand-new draft registration filing, this is not set so you will get an `undefined` value.

When a user has created a NR, is on the My Business Registry page, and clicks "use this NR" for a registration filing, a draft filing is created here: https://github.com/bcgov/sbc-auth/blob/64a61f807537de467af53b5617b8aab030cc0602/auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue#L393

Reply:
I changed it back to draftFiling.registration.businessTypeConfirm || null in my latest commit 674dc35.